types/sdk.js

```diff
@@ -3,7 +3,7 @@
 /* Client */
 export type ClientRequest = {
   uri: string;
-  method: MethodType;
+  method: string;
```

Comment:
Originaly there was a `MethodType` loaded from here:
**v0.46.0 -** https://github.com/facebook/flow/blob/v0.46.0/lib/bom.js#L841

But in the latest version, it got deleted:
**v0.47.0 -** https://github.com/facebook/flow/blob/v0.47.0/lib/bom.js

Also, in the latest release, there is no notice about removed types: https://github.com/facebook/flow/releases/tag/v0.47.0